### PR TITLE
Cleaning docs

### PR DESCRIPTION
**Pull request by antgonza**
In this PR:
- Fixing documentation based on Gail's recommendation.
- Templates -> Information files in documentation

### CODE REVIEW

File: qiita_pet/support_files/doc/source/faq.rst [hunk 1]
```diff
@@ -4,26 +4,25 @@ Frequently Asked Questions
 What kind of data can I upload to Qiita for processing?
 -------------------------------------------------------
 
-We need 3 things: raw data, sample template, and prep template. At this
-moment, raw data is fastq files without demultiplexing with forward,
-reverse (optional) and barcode reads. We should have before the end of
-the week SFF processing so it's OK to upload. Note that we are accepting
```
Comment (ackermag) [inline qiita_pet/support_files/doc/source/faq.rst]: Remove: We should have before the end of the week SFF processing so it's OK to upload.

File: qiita_pet/support_files/doc/source/faq.rst [hunk 2]
```diff
@@ -34,31 +33,32 @@ Example study processing workflow
...
+   to more than one prep information file.
+#. Preprocess your files. For target gene amplicon sequencing, this will demux
+   and QC. There are multiple options for preprocessing depending on the
+   barcode format and the data output from the sequencing center - this may
+   require a series of trial and error to establish the correct option for
+   your data files. After demultiplexing a log file is generated with
+   statistics about the files demultiplexed including the number of sequences
+   assigned per sample.
+#. Process each of your preprocessed data types. For target gene, this will
+   perform close OTU picking against the latest version of Greengenes and can
```
Comment (ackermag) [inline qiita_pet/support_files/doc/source/faq.rst]: "closed" instead of "close"?

File: README.rst [hunk 3]
```diff
@@ -46,6 +46,14 @@ Current features
 * Search over existing studies (see known issues).
 * Generate basic visualizations with the available studies and datasets.
 
+Accepted raw files
+------------------
+
+* Multiplexed SFF
+* Multiplexed FASTQ: forward, reverse (optional), and barcodes
+* Per sample FASTQ: forward
+* Multiplexed fasta/qual files.
```
Comment (ElDeveloper) [inline README.rst]: fasta -> FASTA and perhaps remove the period at the end of this sentence or add it to the other points? ... that is of course a minor suggestion, but yeah I noticed that :neckbeard:

File: qiita_pet/support_files/doc/source/faq.rst [hunk 4]
```diff
@@ -4,26 +4,25 @@ Frequently Asked Questions
...
 
-We need 3 things: raw data, sample template, and prep template. At this
-moment, raw data is fastq files without demultiplexing with forward,
-reverse (optional) and barcode reads. We should have before the end of
-the week SFF processing so it's OK to upload. Note that we are accepting
-any kind of target gene (16S, 18S, ITS, whatever) as long as they have
-some kind of demultiplexing strategy and that you can also upload WGS.
-However, WGS processing is not ready.
+Processing in Qiita requires 3 things: raw data, sample and prep information
+files. `Here <https://github.com/biocore/qiita/blob/master/README.rst#accepted-raw-files>__`
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/faq.rst]: This reads "Here a list" ... maybe it was intended to be "Here you can find" or "Here is".

File: qiita_pet/support_files/doc/source/tutorials/no-raw-sequences.rst [hunk 5]
```diff
@@ -57,8 +57,8 @@ named ``forward`` and all of our reverse reads are in a folder named
     cat forward/*.fastq > forward.fastq
     cat reverse/*.fastq > reverse.fastq
 
-While you don't have to compress the generated files, it will make data
-transfer and storage more convenient, the preferred and only supported
+While there is no requirement to comprass the generated files, it makes data
+transfer and storage more convenient. The preferred and only supported
```
Comment (ackermag) [inline qiita_pet/support_files/doc/source/tutorials/no-raw-sequences.rst]: 'compress' not 'comprass'

File: qiita_pet/support_files/doc/source/faq.rst [hunk 6]
```diff
@@ -4,26 +4,25 @@ Frequently Asked Questions
...
+a list of currently supported raw files files. Note that we are accepting
+any kind of target gene (16S, 18S, ITS, whatever) as long as there is
+some kind of demultiplexing strategy. You can also upload WGS however, WGS
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/faq.rst]: Hmmm the last part is a bit confusing " some kind of demultiplexing strategy" as before you mention that per sample sequence files are acceptable.

File: qiita_pet/support_files/doc/source/faq.rst [hunk 7]
```diff
@@ -4,26 +4,25 @@ Frequently Asked Questions
...
-Sample template is the information about your samples, including
-environmental and other important information about them. The prep
-template is basically what kind of wet lab work all or a subset of the
-samples had. If you collected 100 samples, you are going to need 100
-rows in your sample template describing each of them, this includes
-blanks, etc. Then you prepared 95 of them for 16S and 50 of them for
-18S. Thus, you are going to need 2 prep templates: one with 95 rows
-describing the preparation for 16S, and another one with 50 to
-describing the 18S. For a more complex example go
+Sample information file is the information about the samples, including
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/faq.rst]: What about:

`A sample information file describes the information about ...`

File: qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst [hunk 8]
```diff
@@ -49,57 +55,76 @@ Required fields for centralized Qiita
...
+| ``dna_extracted``                 | ``y`` or ``n``                                                       | Has DNA already been extracted for this sample?                                                                                                                                                                                                                                                                                                   |
++-----------------------------------+----------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (ackermag) [inline qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst]: 'TRUE' or FALSE' I believe

File: qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst [hunk 9]
```diff
@@ -49,57 +55,76 @@ Required fields for centralized Qiita
...
+| ``center_name``                     | free text                                 | Name of the sequencing facility.                                                                                                                                                                               |
++-------------------------------------+-------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``platform``                        | ``Illumina`` or ``LS454``                 | The sequencing technology used in the study. ``Illumina`` sequencing data was generated on an Illumina platform; ``LS454`` sequencing data was generated on a 454 pyrosequencing.                              |
++-------------------------------------+-------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (ackermag) [inline qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst]: insert 'platform' after pyrosequencing

File: qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst [hunk 10]
```diff
@@ -118,15 +143,15 @@ If you are adding target gene data (e.g. 16S, 18S, ITS), there are
...
+| Field name    | Format             | Description                                                                                                                                                                                                    |
++===============+====================+================================================================================================================================================================================================================+
+| ``primer``    | IUPAC characters   | The primer sequence (this is usually the forward primer for Illumina processed data, or the barcoded primer for LS454 data; `examples <http://www.nature.com/ismej/journal/v6/n8/extref/ismej20128x2.txt>`__). |
++---------------+--------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (ackermag) [inline qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst]: That link has no 454 example.

File: qiita_pet/support_files/doc/source/faq.rst [hunk 11]
```diff
@@ -34,31 +33,32 @@ Example study processing workflow
...
-   hesitate to contact us.
-#. You can preprocess your files. For target gene, this means
-   demultiplexing and QC.
-
+   sample\_name, duplicated sample names are not permitted. For a full list of
+   required fields, visit :doc:`tutorials/prepare-information-files`.
+#. Add a prep information file to your study for each data type. The prep
+   information file should contain all the samples in the sample information
+   file or a subset. If you have more than one FASTQ file set (forward,
+   reverse (optional) and barcodes) you will need to add a run_prefix
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/faq.rst]: Maybe worth linking to `Required fields for pre-processing target gene data` when you mention run_prefix.

File: qiita_pet/support_files/doc/source/faq.rst [hunk 12]
```diff
@@ -34,31 +33,32 @@ Example study processing workflow
...
+   column. A prep information file and a QIIME compatible mapping file will
+   be available for download after the prep information file is added
+   successfully.
+#. Upload and link your raw data to each of your prep information files.
+   Depending on your barcoding/sequencing strategy you might need 1 or more
+   raw datas file sets. If you have 2 raw data sets you may have to rename one
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/faq.rst]: datas -> datum? not sure about this one

File: qiita_pet/support_files/doc/source/faq.rst [hunk 13]
```diff
@@ -34,31 +33,32 @@ Example study processing workflow
...
+   will over-write on upload. Note that you can have one FASTQ file set linked
+   to more than one prep information file.
+#. Preprocess your files. For target gene amplicon sequencing, this will demux
+   and QC. There are multiple options for preprocessing depending on the
+   barcode format and the data output from the sequencing center - this may
+   require a series of trial and error to establish the correct option for
+   your data files. After demultiplexing a log file is generated with
+   statistics about the files demultiplexed including the number of sequences
+   assigned per sample.
+#. Process each of your preprocessed data types. For target gene, this will
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/faq.rst]: Just noticed that the first sentence of each of the items in this list summarizes the step itself, would it make sense to make the font bold?

File: qiita_pet/support_files/doc/source/qiita-philosophy/index.rst [hunk 14]
```diff
@@ -13,7 +13,7 @@ environments for each pipeline gives the freedom of adding any pipeline with
 any software dependencies to Qiita. Artifacts, basically any file in the
 system, from raw sequence to contingency tables or even data visualizations,
 permits the system to store any kind of data but also define within each
-pipelines which commands and parameters can applied to them.
+pipelines which commands and parameter can applied to them.
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/qiita-philosophy/index.rst]: parameter -> parameters

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst [hunk 15]
```diff
@@ -6,18 +6,20 @@ EBI submission via Qiita
...
+information visit :doc:`prepare-information-files` and pay most attention to EBI required fields,
 without these **Qiita Admins** will not be able to submit. If you want to submit your data or need
-help send an email to `qiita.help@gmail.com <qiita.help@gmail.com>`__.
+help send an email to `qiita.help@gmail.com <qiita.help@gmail.com>`__. Help will include
+advice on additional fields to add to ensure MiXs compliance.
 
-Note that this kind of submissions are time consuming and need full collaboration from the user.
-Thus, do not wait until the last minute to request help. In general, the best time to request a submission
+Note that submissions are time consuming and need full collaboration from the user.
+Do not wait until the last minute to request help. In general, the best time to request a submission
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: I almost feel like this last part should be made a red banner haha.

File: qiita_pet/support_files/doc/source/tutorials/no-raw-sequences.rst [hunk 16]
```diff
@@ -57,8 +57,8 @@ named ``forward`` and all of our reverse reads are in a folder named
     cat forward/*.fastq > forward.fastq
     cat reverse/*.fastq > reverse.fastq
 
-While you don't have to compress the generated files, it will make data
-transfer and storage more convenient, the preferred and only supported
+While there is no requirement to comprass the generated files, it makes data
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/tutorials/no-raw-sequences.rst]: comprass -> compress

File: qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst [hunk 17]
```diff
@@ -49,57 +55,76 @@ Required fields for centralized Qiita
...
++===================+===============================+===============================================================================================================================================================================================================================================================+
+| ``sample_name``   | free text with restrictions   | Identifies a sample. It is the primary key, must be unique and should match the ones in the sample information file. Allowed characters are alphabetic ``[A-Za-z]``, numeric ``[0-9]``, and periods ``.``. Must match the sample_name in the sample template. |
++-------------------+-------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
 Required fields for EBI submission
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 These are the columns required for successfully submit your data to EBI:
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst]: Just noticed this, not part of this PR but this should be changed to:

`These are teh columns required to successfully submit your data to EBI`

File: qiita_pet/support_files/doc/source/faq.rst (hunk 12 above)
Comment (ackermag) [inline qiita_pet/support_files/doc/source/faq.rst]: I think 'data' implies plural

File: qiita_pet/support_files/doc/source/faq.rst (hunk 13 above)
Comment (ackermag) [inline qiita_pet/support_files/doc/source/faq.rst]: Good suggestion

**Comment by ElDeveloper** [comment]
:+1:

**Comment by ackermag** [comment]
I am done with review

File: qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst (hunk 10 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst]: Do you have one? I'm OK with leaving it like that.
